src/Framework/Telemetry/BuildTelemetry.cs

```diff
@@ -74,6 +74,16 @@ internal class BuildTelemetry : TelemetryBase
         /// </summary>
         public string? Host { get; set; }
 
+        /// <summary>
+        /// True if buildcheck was used.
+        /// </summary>
+        public bool? BuildCheckEnabled { get; set; }
+
+        /// <summary>
+        /// True if Smart Application Control was enabled.
+        /// </summary>
+        public bool? SACEnabled { get; set; }
```

Comment:
Sice we are adding telemetry - I'd realy realy want to get numbers on this.

@baronfel - please scream if you have any concerns with adding this datapoint.